internal/controller/http/v2/devicemanagement.go

```diff
@@ -77,3 +78,17 @@ func (r *deviceManagementRoutes) setFeatures(c *gin.Context) {
 
 	c.JSON(http.StatusOK, v2)
 }
+
+func (r *deviceManagementRoutes) getPowerState(c *gin.Context) {
+	guid := c.Param("guid")
+
+	state, err := r.d.GetPowerState(c.Request.Context(), guid)
+	if err != nil {
+		r.l.Error(err, "http - v2 - getPowerState")
```

Comment:
what is different in the dto for powerstate in v2 vs v1?

Reply:
They're not different but I thought we wanted a v2 for everything in case we want to make small changes easily in the future

Reply:
Going forward, only if there are differences do we create a v2 at this point.